trade-finance-manager-api/src/v1/controllers/deal-add-tfm-data/index.js

```diff
@@ -3,8 +3,9 @@ const generateDateReceived = require('./dateReceived');
 const addDealProduct = require('./dealProduct');
 const addDealPricingAndRisk = require('./dealPricingAndRisk');
 const addDealStage = require('./dealStage');
+const { generatePortalUserInformation } = require("../../helpers/generateUserInformation");
```

Comment:
typo, will update to
```suggestion
const { generatePortalUserInformation } = require("@ukef/dtfs2-common/src/helpers/changeStream/generateUserInformation");
```